src/Bowling.js

```diff
@@ -0,0 +1,54 @@
+function Scorecard() {
+  this.score = 0;
+  this.frames = [];
+  this.currentFrame = [];
+  this.frameCounter = 0;
+};
+
+Scorecard.prototype.rolls = function (roll1, roll2) {
+  this.currentFrame = []
```

Comment:
Do you need to add if statements to account for different scores and outcomes (strikes, etc)?